src/api.rs

```diff
@@ -85,18 +28,18 @@ impl Api {
     pub fn new<C: Into<Config>>(config: C) -> Result<Self, ApiError> {
         let config = config.into();
 
-        let mut builder = ClientBuilder::new();
-        builder = if let Some(proxy) = config.proxy {
-            builder.proxy(proxy)
+        let mut builder = HttpClientBuilder::new();
```

Comment:
BTW, it will be better if user can pass their own builder, so we even can get rid of proxy setup